documentation/modules/security/proc-installing-certs-per-listener.adoc

```diff
@@ -3,21 +3,41 @@
 // assembly-security.adoc
 
 [id='proc-installing-certs-per-listener-{context}']
-= Providing your own Kafka listener certificates
+= Providing your own Kafka listener certificates for TLS authentication
 
-This procedure shows how to configure a listener to use your own private key and server certificate, called a xref:kafka-listener-certificates-{context}[Kafka listener certificate].
+[role="_abstract"]
+Listeners provide client access to Kafka brokers.
+You configure listeners in the `Kafka` resource, including the configuration required for client access using TLS authentication.
 
-Your client applications should use the CA public key as a trusted certificate in order to verify the identity of the Kafka broker.
+By default, the listeners use certificates signed by the internal self-signed CAs generated by Strimzi.
+A public CA certificate is generated by the Cluster Operator when it creates a Kafka cluster.
+When you configure a client for TLS, you add the public CA certificate to its truststore configuration to verify the Kafka cluster.
+You can also xref:installing-your-own-ca-certificates-str[install and use your own CA certificates]. 
+Or you can configure a listener using `brokerCertChainAndKey` properties and use a custom server certificate.
+
+The `brokerCertChainAndKey` properties allow you to authenticate access to Kafka brokers using your own custom certificates at the listener-level.
+You create a secret with your own private key and server certificate, then specify the key and certificate in the listener's `brokerCertChainAndKey` configuration.
+The server certificate must be signed by a CA.
+You add the public CA that signed the server certificate to the client's truststore configuration.
+Custom certificates are not managed by Strimzi, so you need to renew them manually. 
+
+NOTE: Listener certificates are used for server authentication. 
+For client authentication, you add `authentication.type: TLS` to your listener and Kafka user configuration for the client.
+You must also add a public user certificate to the keystore configuration of the client.
+The public user certificate must be signed by the clients CA certificate generated by the Cluster Operator.
+Or you can xref:installing-your-own-ca-certificates-str[install and use your own clients CA]. 
 
 .Prerequisites
 
-* A Kubernetes cluster.
 * The Cluster Operator is running.
-* For each listener, a compatible server certificate signed by an external CA.
-** Provide an X.509 certificate in PEM format.
-** Specify the correct Subject Alternative Names (SANs) for each listener.
+* For each listener, you have the following:
+** A compatible server certificate signed by an external CA. (Provide an X.509 certificate in PEM format.)
+** Subject Alternative Names (SANs) specified in each certificate for each listener.
```

Comment:
I think that quite often the users have one certificate for multiple listeners. Maybe this can be incorporated here? Right, I think it might be read by some as if each listener needs to have a different certificate.

Reply:
Changed to:

> * Each listener requires the following:
> ** A compatible server certificate signed by an external CA. (Provide an X.509 certificate in PEM format.)
> +
> You can use one listener certificate for multiple listeners.
> ** Subject Alternative Names (SANs) specified in the certificate for each listener.
> For more information, see xref:ref-alternative-subjects-certs-for-listeners-{context}[].